src/features/farming/bakery/components/CraftingItems.tsx

```diff
@@ -73,6 +142,17 @@ export const CraftingItems: React.FC<Props> = ({ items }) => {
       </div>
       <OuterPanel className="flex-1 w-1/3">
         <div className="flex flex-col justify-center items-center p-2 relative">
+          {expiryTime && (
+            <>
+              <img src={stopwatch} className="w-4  absolute left-0 -top-4" />
```

Comment:
I think the stopwatch image should be placed inside the span since its colors conflict with the panel's brown colors. What do you think?
<img width="173" alt="Screen Shot 2022-05-31 at 10 22 02 AM" src="https://user-images.githubusercontent.com/18549210/171080135-3363bddf-8908-4c21-8994-e7076bb0d685.png">

Reply:
I agree with this. It also shouldn't wrap on mobile as it covers the item name.
<img width="100" alt="Screen Shot 2022-06-02 at 12 19 00 pm" src="https://user-images.githubusercontent.com/17863697/171535483-5dbb6f17-885f-47c9-8d38-36ad6f00bb0e.png">